src/libproxychains.c

```diff
@@ -86,15 +87,14 @@ static int init_l = 0;
 
 static void get_chain_data(proxy_data * pd, unsigned int *proxy_count, chain_type * ct);
 
-static void* load_sym(char* symname, void* proxyfunc) {
-
+static void* load_sym(char* symname, void* proxyfunc, int is_mandatory) {
 	void *funcptr = dlsym(RTLD_NEXT, symname);
 
-	if(!funcptr) {
-		fprintf(stderr, "Cannot load symbol '%s' %s\n", symname, dlerror());
+	if(is_mandatory && !funcptr) {
+		fprintf(stderr, "Cannot load symbol '%s', is_mandatory=%d, %s\n", symname, is_mandatory, dlerror());
```

Comment:
we don't need the changes here either, it will only be printed for mandatory funcs anyway

Reply:
fixed in latest commit.